packages/flutter_tools/templates/module/ios/library/Flutter.tmpl/podhelper.rb.tmpl

```diff
@@ -93,13 +106,13 @@ def install_flutter_application_pod(flutter_application_path)
 
   # Keep pod and script phase paths relative so they can be checked into source control.
   # Process will be run from project directory.
-  current_directory_pathname = Pathname.new __dir__
-  project_directory_pathname = Pathname.new Dir.pwd
+  current_directory_pathname = Pathname.new File.expand_path('..', __FILE__)
+  project_directory_pathname = defined_in_file.dirname
   relative = current_directory_pathname.relative_path_from project_directory_pathname
   pod '{{projectName}}', :path => relative.to_s, :inhibit_warnings => true
 
   flutter_export_environment_path = File.join('${SRCROOT}', relative, 'flutter_export_environment.sh');
-  script_phase :name => 'Run Flutter Build {{projectName}} Script',
+  script_phase :name => 'Run Flutter Build flutterapp Script',
```

Comment:
is this intentional?

Reply:
No, I meant to revert this.  Good catch!